code/tools/trim_schema_json.py

```diff
@@ -21,7 +21,8 @@ def should_skip_item(site, item):
             if type_value in skip_types:
                 return True
     elif "@type" not in item:
-        return True
+        print(f"Warning: Item without @type field found for site {site}, keeping item: {str(item)[:100]}...")
+        return False
```

Comment:
There is a behavior change here, from returning True in this condition to returning False.  This is intentional that we don't want to skip the item anymore?